Mathlib/Algebra/Module/Presentation/DirectSum.lean

```diff
@@ -17,15 +17,13 @@ a presentation of `ι →₀ M`.
 
 -/
 
-universe w' w₀ w₁ w v u
-
 namespace Module
 
 open DirectSum
 
-variable {A : Type u} [Ring A] {ι : Type w} [DecidableEq ι]
-  (relations : ι → Relations.{w₀, w₁} A)
-  {M : ι → Type v} [∀ i, AddCommGroup (M i)] [∀ i, Module A (M i)]
+variable {A : Type*} [Ring A] {ι : Type*} [DecidableEq ι]
+  (relations : ι → Relations A)
+  {M : ι → Type*} [∀ i, AddCommGroup (M i)] [∀ i, Module A (M i)]
 
```

Comment:
Please don't touch `.{u}`s. This is effectively changing `Type u` to `Type _`, which is a step in the wrong direction IMO.

Reply:
Should I undo all ` .{u}` deletions?

Reply:
Should `[Mathlib/Algebra/Algebra/Basic](https://github.com/leanprover-community/mathlib4/pull/24285/files#diff-ac53643872997840bf71ba1e821cd4e1b63add23bf12ad135e37e121900da9ca)` be reverted as well? Afaik the universe `v` was not used anywhere except `.{v+1}`  that I deleted.

Reply:
And should [Mathlib/Algebra/Module/Presentation/Differentials.lean](https://github.com/leanprover-community/mathlib4/pull/24285/files#diff-ec6b6e59a4cb03ff8b54d1a509148d5f4199866d69d0774a30c65fe655d03af7) be reverted as well?

Reply:
Yes I believe so. As I said, going from `Type u` to `Type _` is the wrong direction even if the universe only appears once. For `Differentials.lean` the changes to `Type*` are good and should be kept.

Reply:
OK, done.